lib/EventSystemEpic/ProjectFeature/Model/ignore.js

```diff
@@ -0,0 +1,12 @@
+"use babel";
+// @flow
+
+const nodeModules = "node_modules/";
+const tmpDir = "tmp/";
+const tmpExt = ".tmp";
+const gitExt = ".git";
+
+const paths = [nodeModules, tmpDir, tmpExt, gitExt];
+
+export const ignoreFilePaths = (path: ?string) =>
+  paths.reduce((res, test) => !(path && path.includes(test)), true);
```

Comment:
Why using a reducer when you can use array.include ?

```suggestion
  paths.includes(path);
```

Reply:
I want to test whether or not the content of the path includes one of the values in paths, what you're suggesting doesn't work